x-pack/plugins/canvas/public/components/hooks/workpad/use_download_workpad.ts

```diff
@@ -34,7 +34,7 @@ export const useDownloadWorkpad = () => {
   return useCallback(
     async (workpadId: string) => {
       try {
-        const workpad = await workpadService.get(workpadId);
```

Comment:
Same thing as the comment above -- this ID doesn't originate from a deep link, so you don't need to resolve the workpad here, you just need to get it directly.